### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by chrisguttandin**
This pull request adds to missing words and simplifies an if statement.

Please let me know if there is anything I should change.

***
<a href="https://pr-preview.s3.amazonaws.com/chrisguttandin/web-audio-api/pull/2118.html" title="Last updated on Jan 24, 2020, 9:57 PM UTC (7b227c8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WebAudio/web-audio-api/2118/21bad4b...chrisguttandin:7b227c8.html" title="Last updated on Jan 24, 2020, 9:57 PM UTC (7b227c8)">Diff</a>

### CODE REVIEW

**Comment by hoch** [comment]
Chris, please take a look at ipr detail.

File: index.bs [hunk 1]
```diff
@@ -4983,7 +4983,7 @@ function process(numberOfFrames) {
 	// Determine loop endpoints as applicable
 	let actualLoopStart, actualLoopEnd;
 	if (loop && buffer != null) {
-		if (loopStart >= 0 && loopEnd > 0 && loopStart < loopEnd) {
+		if (loopStart >= 0 && loopEnd > loopStart) {
```
Comment (rtoy) [inline index.bs]: What's wrong with the original?

Comment (chrisguttandin) [reply index.bs]: Nothing. `loopEnd > 0` is just redundant.

**Comment by chrisguttandin** [comment]
@hoch Unfortunately I can't use the "Repository Manager" app. I linked my accounts but now it can't load the pull request anymore. The error I get says "Failed to fetch".

Do you have any idea how I could resolve this?

File: index.bs (hunk 1 above)
Comment (rtoy) [reply index.bs]: `'loopEnd` and `loopStart` can both be negative, as mentioned in https://webaudio.github.io/web-audio-api/#dom-audiobuffersourcenode-loopend and https://webaudio.github.io/web-audio-api/#dom-audiobuffersourcenode-loopstart.

As reference code, I prefer to keep to keep things explicit.

**Comment by rtoy** [comment]
This is a non-substantive change since it's just fixing a few typos.  I can fix that.  Don't know how to fix the issue about failing to load the PR anymore.

**Comment by rtoy** [comment]
Marked as non substantive for IPR from ash-nazg.

File: index.bs (hunk 1 above)
Comment (rtoy) [reply index.bs]: I defer to the editors for the final decision.

Comment (chrisguttandin) [reply index.bs]: Yes sure, I'm happy to role this back if the editors think it makes more sense the other way round.

Comment (hoch) [reply index.bs]: Yes, Chris - I agree with @rtoy on this. As a spec document, we should aim clarify. Could you please fix it?

Thanks for your contribution as always!

Comment (chrisguttandin) [reply index.bs]: Alright, I rolled it back.

**Pull request merged**